packages/esm-ugandaemr-app/src/createDashboardLink.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useMemo } from 'react';
+import { ConfigurableLink } from '@openmrs/esm-framework';
+import { BrowserRouter, useLocation } from 'react-router-dom';
+
+export interface DashboardLinkConfig {
+  name: string;
+  title: string;
+}
+
+function DashboardExtension({ dashboardLinkConfig }: { dashboardLinkConfig: DashboardLinkConfig }) {
```

Comment:
i don't think u need this. There slots available and u could just attach them the same way i add links to the patient chart.

Reply:
this was removed in subsequent prs